coilsnake/model/common/ips.py

```diff
@@ -74,5 +75,47 @@ def is_applied(self, rom):
                     return False
         return True
 
-    def create(self, clean_rom, modified_rom):
-        raise NotImplementedError("IPS patch creation is not implemented yet.")
+    def create(self, clean_rom, hacked_rom, patch_path):
+        """Creates an IPS patch from the source and target ROMs."""
+
+        # Create the records.
+        i = None
+        records = {}
+        cr = open(clean_rom, "rb")
```

Comment:
You should use a context manager for `cr`, `hr`, and `pfile`.

E.g. 
```python
with open(clean_rom, "rb") as cr, open(hacked_rom, "rb") as hr:
    # do stuff with `cr` and `hr`
```

This will ensure that they are closed properly even if an error occurs somewhere!